assets/in_page_search.js

```diff
@@ -0,0 +1,141 @@
+class InPageSearch {
```

Comment:
Wow, JavaScript has changed since the days i was a frontend hacker...

Reply:
Mind you, these are not real classes: it's just syntactic sugar over the usual prototype inheritance of JS.
JS still has no notion of object in the sense of OOP, so it's impish as usual. :laughing: